packages/nimble-components/src/banner/tests/banner-matrix.stories.ts

```diff
@@ -0,0 +1,100 @@
+import type { Story, Meta } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html, ViewTemplate, when } from '@microsoft/fast-element';
+import {
+    createMatrixThemeStory,
+    createStory
+} from '../../utilities/tests/storybook';
+import {
+    createMatrix,
+    sharedMatrixParameters
+} from '../../utilities/tests/matrix';
+import { hiddenWrapper } from '../../utilities/tests/hidden';
+import '../../all-components';
+import { loremIpsum } from '../../utilities/tests/lorem-ipsum';
+
+const metadata: Meta = {
+    title: 'Tests/Banner',
+    decorators: [withXD],
+    parameters: {
+        ...sharedMatrixParameters(),
+        design: {
+            artboardUrl:
+                'https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/29c405f7-08ea-48b6-973f-546970b9dbab/specs/'
+        }
+    }
+};
+
+export default metadata;
+
+const severityStates = [
```

Comment:
We should dynamically build this set of states from the type.

Reply:
Done